mail/send.go

```diff
@@ -27,6 +29,14 @@ type loginAuth struct {
 	username, password string
 }
 
+type MailReport struct {
+	Policy       string                `bigquery:"policyUid"`
+	Name         string                `bigquery:"name"`
+	Email        string                `bigquery:"email"`
+	CreationDate bigquery.NullDateTime `bigquery:"creationDate"`
+	MailError    string                `bigquery:"mailError"`
+}
+
```

Comment:
Move me to mail/models.go

Cleanup key names. Since we are doing something from scratch lets make them all coincide (Name in Go with name in BigQuery). If needed we can update the schema, not a big deal.

Also make it so that the use is clear. In the example above it doesn't seem so:
```golang
report := MailReport{policy, sender, recipients, date, message}
```

Reply:
The new struct is: 

```
type MailReport struct {
	Policy         string                `bigquery:"policyUid"`
	SenderName     string                `bigquery:"senderName"`
	RecipientEmail string                `bigquery:"recipientEmail"`
	CreationDate   bigquery.NullDateTime `bigquery:"creationDate"`
	MailError      string                `bigquery:"mailError"`
}
```

@diogo-wopta update/recreate bigquery table keeping in mind that now 
`MailReportCollection         string = "mailReport"`

Reply:
@fpittoni table created on bigquery

Reply:
I tested it and it works. Don't forget to delete the old mail-report table.